opus/application/apps/cart/views.py

```diff
@@ -120,13 +120,21 @@ def api_view_cart(request):
     info = _get_download_info(product_types, session_id)
     count, recycled_count = get_cart_count(session_id, recycled=True)
 
-    for name, details in info['product_cat_list']:
-        for type in details:
-            if (type['slug_name'] in not_selected_product_types or
-                not type['default_checked']):
-                type['selected'] = ''
-            else:
-                type['selected'] = 'checked'
+    for name, product_versions in info['product_cat_dict'].items():
+        for ver, types in product_versions.items():
+            for type in types:
```

Comment:
`type` is the name of a Python built-in function and should not be used as the name of a variable.